cumulusci/tasks/salesforce/ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,36 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ActivateProcessBuilderProcesses(BaseSalesforceApiTask):
+    """
+    Create a Salesforce Activation Process Builder which takes in a list as options and
+    runs flows only on those listed DeveloperName's.
```

Comment:
+1 for docstring, I'm just going to propose an edit for clarity.

"""Activate the Flows with the supplied Developer Names"""

Reply:
Implemented